CMakeLists.txt

```diff
@@ -48,8 +48,13 @@ endif()
 
 option(CUB_ENABLE_HEADER_TESTING "Test that all public headers compile." ON)
 option(CUB_ENABLE_TESTING "Build CUB testing suite." ON)
+set(CUB_TEST_VARIANT "ALL" CACHE STRING "Specify which CUB test variants should be built (options: ALL, THOROUGH, BENCHMARK, MINIMAL).")
```

Comment:
Can we make this a series of booleans

```
CUB_TEST_VARIANT_THOROUGH
CUB_TEST_VARIANT_MINIMAL
CUB_TEST_VARIANT_BENCHMARK
```

that default to `ON`? It would allow us to just turn off `THOROUGH` variants but leave still have some testing + benchmarks.

Reply:
Done.